changelog/issue-895-1.md

```diff
@@ -0,0 +1,3 @@
+level: silent
+reference: issue 895
+---
```

Comment:
Let's remove this changelog even if CI test fail. We'll merge it to the material-ui pr which already has a changelog.